models/correlation_detector.h

```diff
@@ -100,9 +104,8 @@ the
 delta_tau            ms       Bin width. This has to be an odd multiple of
                               the resolution, to allow the symmetry between
                               positive and negative time-lags.
```

Comment:
I would mention the default value also here, also the default for tau_max (that one below in the tau_max part)

Reply:
Done for all changed models in fb8470296.